docs/machineset-aws.md

```diff
@@ -1,19 +1,36 @@
 # Creating an AWS Windows MachineSet
 
-_\<windows_container_ami\>_ should be replaced with the AMI ID of a Windows
-image with the Docker container run-time installed. You must use Windows Server
-2019 with a version 10.0.17763.1457 or earlier to work around Windows containers
-behind a Kubernetes load balancer becoming unreachable
-[issue](https://github.com/microsoft/Windows-Containers/issues/78).
-                                                                           
-_\<infrastructureID\>_ should be replaced with the output of:
+Find enclosed a sample YAML, that defines a Windows MachineSet object running on AWS that
+the Windows Machine Config Operator (WMCO) can react upon.
+
+## Parameters
+
+`<infrastructureID>` should be replaced with the infrastructure ID that is based on the cluster ID that
+you set when you provisioned the cluster.
+
+To obtain the infrastructure ID run the following command:
 ```shell script
- oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 ```
-_\<region\>_ should be replaced with a valid AWS region like `us-east-1`.
-_\<zone\>_ should be replaced with a valid AWS availability zone like `us-east-1a`.
 
+`<region>` should be replaced with a valid AWS region like `us-east-1`.
+
+`<zone>` should be replaced with a valid AWS availability zone like `us-east-1a`.
+
```

Comment:
Please remove these extra lines you are introducing in between. This is again introducing inconsistencies between the docs.

Reply:
@aravindhp   as per [the master branch](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-aws.md) the parameters are mixed in the same line, which makes them difficult to read. 

The proposed "extra lines" help to easily identify them in the document. In addition, the [machineset-azure.md](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-azure.md) document  entertains an itemized list of parameters, that was used as the target reference for this change.

Reply:
Why given we have not done so anywhere else? This seems like some GitHub markdown foo is needed.

Reply:
Reverted to the main objective, the "network requirement in vSphere". Will follow-up with the formatting issues in a low low priority PR.

Thanks @aravindhp  for pulling back.